packages/create-sitecore-jss/src/templates/nextjs/src/pages/[[...path]].tsx

```diff
@@ -29,9 +30,11 @@ const SitecorePage = ({ notFound, componentProps, layoutData }: SitecorePageProp
     return <NotFound />;
   }
 
+  const isEditing = layoutData.sitecore.context.pageEditing;
```

Comment:
We have [isEditorActive](https://github.com/Sitecore/jss/blob/dev/packages/sitecore-jss/src/utils/editing.ts#L67) utility for such purposes